index.html

```diff
@@ -86,7 +77,12 @@ <h2>Leave a Message</h2>
     <h2>Messages</h2>
     <ul></ul>
   </section>
-  <footer></footer>
+  <footer>
+    <section id="connect">
+
+      <h2>Connect</h2><a href="https://github.com/Jessica12492/intro-to-programming-dorado">Github</a><a
```

Comment:
Right now, the font sizes of this text looks okay to me on all screen sizes; but if you wanted to change it to be smaller on mobile and tablet; you could add a style rule to your existing media queries on lines 283, 308 and 331. for example:

```css
/* media query for mobile Phones */
@media only screen and (max-width: 480px) {

    section,
    body {

        flex-wrap: wrap;

        background: grey;
        border: 20px;

    }

    header ul {
        font-size: 4px;
        margin: 5px;
        padding: 2px;

    }

    header ul li {
        padding: 0;
    }

    #connect a {
      font-size: 5px;
    }

}
```